src/test/java/org/calrissian/flowmix/api/aggregator/CountAggregatorTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 Calrissian.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.calrissian.flowmix.api.aggregator;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ * @author Miguel A. Fuentes Buchholtz
```

Comment:
I didn't notice this before. This isn't something we generally do in open source projects in Calrissian. It's also not something we tend to do in Apache (which is where our licensing is based). The code is owned by all the members of the project and finding out the author of certain changes can easily be done in git through annotating the changes to a specific file. Putting an author on a file demonstrates more narrow ownership than the wider community and that's not generally accepted.